### DISCUSSION TITLE
How to update/replace buggy NWB files?

### DISCUSSION THREAD

**Discussion by akgillespie8** [dandi/helpdesk#53] [Q&A]
Hi all, 
We unfortunately recently found a bug in our NWB conversion pipeline and so have needed to re-convert a bunch of files, including some that I've already published as a Dandiset (000115). Is there a way to update/replace these files while maintaining the same dandiset ID/doi? Thanks!

**Comment by jwodder** [comment]
Published files, by design, cannot be changed, but you can publish a new version of the same Dandiset.  If you upload the new files the same way you did the originals, they will keep the same Dandiset ID and will replace the ones available under the "draft" version of the Dandiset, but the files under the published version of the Dandiset will be unchanged.  You can then publish a new version of the Dandiset, and this new version will have a new DOI.
